wf-modules/jboss/container/wildfly/standalone-conf/java/added/standalone.conf

```diff
@@ -0,0 +1,15 @@
+
+source /usr/local/dynamic-resources/dynamic_resources.sh
+export GC_MAX_METASPACE_SIZE=${GC_MAX_METASPACE_SIZE:-256}
+JAVA_OPTS="$(adjust_java_options ${JAVA_OPTS})"
+
+# Make sure that we use /dev/urandom (CLOUD-422)
+JAVA_OPTS="${JAVA_OPTS} -Djava.security.egd=file:/dev/./urandom"
```

Comment:
Can't the `-Djava.security.egd` value be changed to `file:/dev/urandom`? (IOW is there a reason why it needs to be written as is?)

Reply:
Yes, there used to be a reason in JDK 8 and earlier since it handled the value differently. Hence being conservative here and using the previous value that does not harm, but I've introduced https://issues.redhat.com/browse/KEYCLOAK-12828 to revisit this eventually

Reply:
Ok, thanks for filing that (didn't know the current form was suggested JDK 8 workaround).